tests/console/validate_raid.pm

```diff
@@ -95,14 +93,6 @@ sub prepare_test_data {
             $vfat_efi,
             $btrfs, $swap,
         );
-        if (is_sle('>=15-SP2') || is_sle('=12-SP5') ||
-            (is_tumbleweed() && !check_var('FLAVOR', 'NET'))) {
-            @raid = ($raid_both_same_level, @raid_detail);
-        }
-        else {
-            # raid0 for swap is required in some products
-            @raid = (($raid_level, $raid0), @raid_detail);
-        }
```

Comment:
it happen to me as well that when I move the final `else` inside I missed that `@partitioning` and `@raid` do are not declared as default values. So as the code is now, you would miss to test raid on aarch.

Reply:
Ah, ok. I reverted the code to the old state, but didn't notice the issue. Thanks for pointing to it. I've defined `@raid` for aarch64.

Reply:
The new VR: https://openqa.opensuse.org/t1050993